src/main/scala/scalismo/image/DiscreteScalarImage.scala

```diff
@@ -31,7 +31,7 @@
 // * @tparam D  The dimensionality of the image
 // * @tparam A The type of the pixel (needs to implement Scalar).
 // */
-//class DiscreteScalarImage[D: NDSpace, A: Scalar: ClassTag](override val domain: DiscreteImageDomain[D],
+//class DiscreteImage[D: NDSpace, A: Scalar: ClassTag](override val domain: DiscreteImageDomain[D],
```

Comment:
remove commented code? (also a lot following in the same file)

Reply:
I will delete all the code that was commented out.  In most cases I could just remove the source file itself.